includes/Story_Post_Type.php

```diff
@@ -200,7 +200,7 @@ public static function show_admin_bar( $show ) {
 	 * Highjack editor with custom editor.
 	 *
 	 * @param bool    $replace Bool if to replace editor or not.
-	 * @param WP_Post $post    Current post object.
+	 * @param WP_Post $post Current post object.
```

Comment:
Unrelated change here.

(NB: doesn't PHPCS cover this? 🤔)

Reply:
PHPCS went weird. https://github.com/google/web-stories-wp/pull/175/commits/ba7d5fc86d484dee22161941caf619a6432f414b